rxcomponents/src/test/java/com/redspace/utils/rxcomponents/ComponentsTests.kt

```diff
@@ -0,0 +1,57 @@
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+
+object ComponentSpec : Spek({
+
+    describe("When waiting for a component") {
+        val testSubject = ComponentManager<Unit>()
+        val obs = testSubject.component.test()
+
+        it("should not emit anything") {
+            obs.assertNoValues().assertNotComplete().assertNoErrors()
```

Comment:
Seems like these 3 assertions could be collapsed to one

Reply:
I can use `assertNotTerminated` since it's from a single